payments_service/braintree/templates/braintree/emails/subscription_header.html

```diff
@@ -10,6 +10,8 @@
   <body id="email">
     <main>
       <div id="wrapper">
-        <div id="logo"></div>
-        <h1>{{ seller.name }}</h1>
-        <p class="user-email">placeholder@placeholder.com</p>
+        <div id="padding">
```

Comment:
padding is not a great id name as it's really unclear looking at the CSS what it refers to.